docs/plugins/testing_workshop_docs/Pythons-assert-keyword.md

```diff
@@ -0,0 +1,80 @@
+This article will be more valuable if you are familiar with the Python programming language and Napari software. It is the first in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 5:23. This article is a summary of the information in the video and should stand on its own. The other articles are:  
+Article 1: This article  
+Article 2: [Pytest testing framework](./Pytest-testing-frameworks)  
+Article 3: [Readers and fixtures](./Readers-and-fixtures)  
+Article 4: [Test coverage](./Test-Coverage)  
+Article 5: Testing widgets  
+  
+This article covers:  
+[Test for Pass](#test-for-pass)  
+[Test for Fail](#test-for-fail)  
+
+Resources
+The example plugin and all the tests discussed in this article are available in [this GitHub repository](https://github.com/DragaDoncila/plugin-tests).
+  
+The key to testing in Python is the [assert](https://docs.google.com/presentation/d/1RFja0o6cZ8lAalAve8heuJ-Lrb4nOSUnfdpOSEhqqNo/edit#slide=id.g107356847e4_0_22) keyword. We *assert* a Boolean expression is true and create an error message that appears when that expression is false. 
+    
+    `assert <Boolean expression>, <error message>`  
```

Comment:
I put the back-ticks around everything that looked like code to draw attention to it.  I think it makes it more scan-able when someone is looking for a command.  I don't mind taking them out but, won't it create an inconsistency?

Reply:
Please keep the back-ticks on all the code looking things! I only mention it in this case because (I think - I may have missed another example) this is the only case of an indented block that also has back ticks. If you want to keep them it's no problem! I just though that perhaps it was by accident. I'll try demonstrate in case that makes it clearer. I thought you intended case 2, but had written case 3.

## 1. back-ticked code inline with text
Here is some inline `code`. It needs the back ticks 100%.

## 2. indented code (a code block)
Here is some

    code

It does not need back ticks, you could put it in if you want

## 3. indented code with ticks
Here is some

    `code`

And now it is indented and has back ticks

Reply:
Thanks.  I corrected it and will upload the new version.

Reply:
Thank you.